docs/pipelineruns.md

```diff
@@ -1182,17 +1182,44 @@ For more information, see the [`LimitRange` support in Pipeline](./compute-resou
 
 ### Configuring a failure timeout
 
-You can use the `timeouts` field to set the `PipelineRun's` desired timeout value in minutes.  There are three sub-fields than can be used to specify failures timeout for the entire pipeline, for tasks, and for `finally` tasks.
+You can use the `timeouts` field to set the `PipelineRun's` desired timeout value in minutes.
+There are three sub-fields:
+- `pipeline`: specifies the timeout for the entire PipelineRun. Defaults to to the global configurable default timeout of 60 minutes.
+When `timeouts.pipeline` has elapsed, any running child TaskRuns will be canceled, regardless of whether they are normal Tasks
+or finally Tasks, and the PipelineRun will fail.
+- `tasks`: specifies the timeout for the cumulative time taken by non-finally Tasks specified in `pipeline.spec.tasks`.
+To specify a timeout for an individual Task, use `pipeline.spec.tasks[].timeout`.
+When `timeouts.tasks` has elapsed, any running child TaskRuns will be canceled, finally Tasks will run if `timeouts.finally` is specified,
+and the PipelineRun will fail.
+- `finally`: the timeout for the cumulative time taken by finally Tasks specified in `pipeline.spec.finally`.
+(Since all finally Tasks run in parallel, this is functionally equivalent to the timeout for any finally Task.)
+When `timeouts.finally` has elapsed, any running finally TaskRuns will be canceled,
+and the PipelineRun will fail.
+
+For example:
 
 ```yaml
 timeouts:
   pipeline: "0h0m60s"
   tasks: "0h0m40s"
   finally: "0h0m20s"
 ```
+
 All three sub-fields are optional, and will be automatically processed according to the following constraint:
 * `timeouts.pipeline >= timeouts.tasks + timeouts.finally`
 
+The global default timeout is set to 60 minutes when you first install Tekton. You can set
+a different global default timeout value using the `default-timeout-minutes` field in
+[`config/config-defaults.yaml`](./../config/config-defaults.yaml).
+
+Each `timeout` field is a `duration` conforming to Go's
+[`ParseDuration`](https://golang.org/pkg/time/#ParseDuration) format. For example, valid
+values are `1h30m`, `1h`, `1m`, and `60s`.
+
+If any of the sub-fields are set to "0", there is no timeout for that section of the PipelineRun,
+meaning that it will run until it encounters an error.
```

Comment:
The other way,
> meaning that it will run until it encounters an error.

meaning that it will run until it completes successfully or encounters an error.